lib/shopify_api/rest/customer.ex

```diff
@@ -26,8 +26,14 @@ defmodule ShopifyAPI.REST.Customer do
       iex> ShopifyAPI.REST.Customer.get(auth, integer)
       {:ok, %{} = customer}
   """
-  def get(%AuthToken{} = auth, customer_id, params \\ []),
-    do: REST.get(auth, "customers/#{customer_id}.json", params)
+  def get(%AuthToken{} = auth, customer_id, params \\ [], options \\ []) do
+    REST.get(
+      auth,
+      "customers/#{customer_id}.json",
+      params,
+      Keywords.merge([pagination: :none], options)
```

Comment:
```suggestion
      Keyword.merge([pagination: :none], options)
```

Reply:
That'll learn me to make a PR without running the tests locally... (still having issues with this project and `asdf`)